goal_src/jak2/game.gp

```diff
@@ -3987,13 +3987,13 @@
 (goal-src-sequence
   ""
   :deps ("$OUT/obj/los-control.o")
-  "levels/common/ai/ai-task-h.gc"
-  "levels/common/ai/bot-h.gc"
-  "levels/common/ai/bot.gc"
-  "levels/common/ai/bot-states.gc"
-  "levels/undefined/hal-h.gc"
-  "levels/undefined/hal-task.gc"
-  "levels/undefined/hal.gc"
+  ;; "levels/common/ai/ai-task-h.gc"
+  ;; "levels/common/ai/bot-h.gc"
+  ;; "levels/common/ai/bot.gc"
+  ;; "levels/common/ai/bot-states.gc"
+  ;; "levels/undefined/hal-h.gc"
+  ;; "levels/undefined/hal-task.gc"
+  ;; "levels/undefined/hal.gc"
```

Comment:
feel free to just remove these - I don't think there's much value in keeping them here. In the `.gd` file it says which `.gc` files go in each `.DGO` already.